content/docs/guides/proxy-exporter.md

```diff
@@ -0,0 +1,375 @@
+* the exporter subsequently starts the scrape after getting Prometheus’ GET requests and once it is done with scraping.
+
+This pattern is only used for certain exporters, namely the [blackbox](https://github.com/prometheus/blackbox_exporter), the [SNMP](https://github.com/prometheus/snmp_exporter) and the [IPMI exporter](https://github.com/soundcloud/ipmi_exporter).
+
+The reason is that we either can’t run the exporter on the targets, e.g. network gear speaking SNMP or servers whose OS we don’t have access too speaking IPMI, or that we are explicitly interested in the distance, e.g. latency and reachability of a website from a specific point outside of our network.
+
+## Running proxy exporters
+
+Proxy exporters are flexible regarding their environment and can be run in many ways. As regular programs, in containers, as background services, on baremetal, virtual machines. Because they are queried and do query over network they do need appropriate open ports. Otherwise they are frugal.
+
+Now let’s try it out for yourself!
+
+Use [Docker](https://www.docker.com/) to start a blackbox exporter container running by running this in a terminal:
+
+```bash
+docker run -p 9115:9115 prom/blackbox-exporter
+```
+
+You should see a few log lines and if everything went well the last one should report `msg="Listening on address"` as seen here:
+
+```
+level=info ts=2018-10-17T15:41:35.4997596Z caller=main.go:324 msg="Listening on address" address=:9115
+```
+
+## Basic querying of proxy exporters
+
+There are two ways of querying:
+
+1. Querying the exporter itself. It has its own metrics, usually available at `/metrics`.
+1. Querying the exporter to scrape another target. Usually available at a "descriptive" endpoint, e.g. `/probe`.
+
+You can manually try the first query type with curl in another terminal. If you don’t have curl, you can also copy the part between the `"` into a browser’s address bar:
+
+<a name="query-exporter"></a>
+
+```bash
+curl "localhost:9115/metrics"
+```
+
+The response should be a list of metrics of mostly starting with `go_` and `process_` in the Prometheus [format](/docs/instrumenting/exposition_formats/#text-format-example).
+
+For the second type of querying we need to provide a target and module as parameters in the HTTP GET Request. The target is a URI or IP and the module must defined in the exporter’s configuration. The blackbox exporter container comes with a meaningful default configuration.  
+We will use the target `prometheus.io` and the predefined module `http_2xx`. It tells the exporter to make a GET request like a browser would if you go to `prometheus.io` and to expect a [200 OK](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#2xx_Success) response.
+
+You can now tell your blackbox exporter to query `prometheus.io` in the terminal with curl:
+
+```bash
+curl "localhost:9115/probe?target=prometheus.io&module=http_2xx"
```

Comment:
I'd add an indication of why you have the quotes here, this trips people up

Reply:
Before I made the PR I tested this guide thoroughly with someone completely new to the command line, containers, Prometheus and exporters. They did not stumble over this, hence I do not worry too much.

Reply:
I am torn; intermediate users might get confused where new ones just type things without questioning it. OTOH, this is not a shell guide. A quick note that quoting protects the URL's special chars can't hurt, though. Also, you should probably use `'`, not `"`.

Reply:
I tested the whole guide on someone who is very new to the shell, containers, Prometheus and exporters and this did not came up as a tripping hazard. I am not overly worried about this.

Reply:
I changed the `"` to `'`. If I put a note in for the `'` I should likely remove them further up at the first invocation of curl with no special signs. And I think that make things more confusing, and there is already enough content people need to learn in this guide.